### ISSUE TITLE
Game mode - US4: Game mode fairly selects randomised actions

### ISSUE THREAD

**Issue by Andre-Pham** [TalkbackTutorial/Application#74]
As a user, I want the actions I'm asked during the game mode to feel randomised and fair so that I can ensure my learning covers the entire variety of Talkback actions.

The game mode tasks the user to perform gestures for points. Hence an important aspect is what actions the user is asked to perform.

The gestures the user is asked to do should be equally distributed. Aspects to consider:
* Opening Talkback menu has three gestures, but this doesn't mean it should be 3x as likely, since the user will have one preference they will use every time. This means gestures can't just be randomly picked from the set available.
* Gestures shouldn't be asked repeatedly. The game shouldn't be able to ask the user to "Scroll down" twice in a row - that's pointless. It also shouldn't be able to repeatedly switch between two actions. There should be a threshold amount of different gestures the user performs before they can receive a repeat action.
* A simple approach is getting all the possible actions, and then randomising them in a queue. Something to consider is that every action will need to be mapped to the gestures that perform it.

**Comment by Andre-Pham** [comment]
Resolved using `GesturePool`.
